frontend/src/helpers/utils/get-element-height.ts

```diff
@@ -0,0 +1,14 @@
+export const getElementHeight = (element: Element | null): number => {
+  if (!element) return 0;
+
+  let elementHeight = element?.clientHeight || 0;
+  elementHeight +=
+    parseInt(window.getComputedStyle(element).getPropertyValue('margin-top')) ||
+    0;
+  elementHeight +=
+    parseInt(
+      window.getComputedStyle(element).getPropertyValue('margin-bottom'),
+    ) || 0;
+
+  return elementHeight;
+};
```

Comment:
Can you add some explanation? Because it's not that intuitive why we should consider the margins :thinking:

Reply:
Because "Search" title has top,bottom margins, and without them, title height is only 32.2px, but the real height is 92px

![image](https://user-images.githubusercontent.com/49813216/188264058-27a35394-f0bd-4da1-a8d0-30d41c25d177.png)

Reply:
До речі... У JS справді немає способу отримати висоту елементу включно з його margin'ами. Але варто бути уважним та зважати що `getElementHeightWithMargins` може поломатися якщо висота елемента вказана явно (JS такий JS :clown_face:)

Demo - https://playcode.io/954066/

Reply:
Yeah, also this "separately scrolling" feature strongly depends of html elements above. And after every adding/deleting new htmls I need to add/delete their height in useLayoutEffect to correctly calculate formTop position. I hope it'll be rarely...